api/src/pcapi/routes/backoffice/filters.py

```diff
@@ -502,10 +503,48 @@ def format_offer_status(status: offer_mixin.OfferStatus) -> str:
             return "Épuisée"
         case offer_mixin.OfferStatus.INACTIVE:
             return "Inactive"
+        case offer_mixin.CollectiveOfferStatus.ARCHIVED:
+            return "Archivée"
         case _:
             return status.value
 
 
+def format_offer_displayed_status(displayed_status: educational_models.CollectiveOfferDisplayedStatus) -> str:
```

Comment:
On s'y retrouvera mieux avec `collective` dans le nom puisque c'est spécifique : 
```suggestion
def format_collective_offer_displayed_status(displayed_status: educational_models.CollectiveOfferDisplayedStatus) -> str:
```